src/components/notes/editor.jsx

```diff
@@ -134,6 +126,17 @@ const Editor = translate()(
       [props.returnUrl, doc]
     )
 
+    // Failure in loading the note ?
+    useEffect(() => {
+      if (!loading && !doc) {
+        // eslint-disable-next-line no-console
+        console.warn(`Could not load note ${noteId}`)
+        if (returnUrl) {
+          document.location = returnUrl
+        }
+      }
+    })
```

Comment:
Should this effect have a dependency on `returnUrl` ?

Reply:
Yep, not a big performance change, but you're right. Fixed